com.unity.render-pipelines.universal/Documentation~/rendering/deferred-rendering-path.md

```diff
@@ -0,0 +1,422 @@
+
+### <a name="render-passes"></a>Deferred Rendering Path render Passes
+
+The following table shows the sequence of Render Pass events in the Deferred Rendering Path.
+
+<table>	
+	<thead>
+	<tr>
+		<th>Render Pass events</th>
+		<th>Deferred Rendering Path Passes</th>
+		<th>SSAO Renderer Feature Passes</th>
+	</tr>
+	</thead>
+	<tbody>
+	<tr>
+		<td>BeforeRendering</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>BeforeRenderingShadows</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>AfterRenderingShadows</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>BeforeRenderingPrePasses</td>
+		<td>Depth, or depth and normal prepass (Forward only materials)</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>AfterRenderingPrePasses</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>BeforeRenderingGbuffer</td>
+		<td>G-buffer Pass (GBufferPass)</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>&#160;</td>
+		<td>Copy G-buffer depth texture</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>AfterRenderingGbuffer</td>
+		<td>&#160;</td>
+		<td>SSAO (optional)</td>
+	</tr>
+	<tr>
+		<td>BeforeRenderingDeferredLights</td>
+		<td>&#160;</td>
+		<td rowspan="4"><img src="../Images/rendering-deferred/decorator-or.png" /></td>
+	</tr>
+    <tr>
+		<td></td>
+		<td>Deferred rendering (stencil)</td>		
+	</tr>
+	<tr>
+		<td>AfterRenderingDeferredLights</td>
+		<td>&#160;</td>		
+	</tr>
+	<tr>
+		<td>BeforeRenderingOpaques</td>
+		<td>Opaque Forward-only Materials</td>
+	</tr>
+	<tr>
+		<td>AfterRenderingOpaques</td>
+		<td>&#160;</td>
+		<td>SSAO and blending (optional)</td>
+	</tr>
+	<tr>
+		<td>BeforeRenderingSkybox</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>AfterRenderingSkybox</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>BeforeRenderingTransparents</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>AfterRenderingTransparents</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>BeforeRenderingPostProcessing</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>AfterRenderingPostProcessing</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tr>
+		<td>AfterRendering</td>
+		<td>&#160;</td>
+		<td>&#160;</td>
+	</tr>
+	<tbody>
+</table>
+
+The following sections describe the Deferred Rendering Path render Passes.
+
+#### Depth, or depth and normal prepass
+
+In the Deferred Rendering Path, in the depth prepass or depth and normal prepass, Unity renders only the Materials that do not support the deferred rendering model. For example, Materials using the Complex Lit shader are such Materials.
+
+In the Deferred Rendering Path, Unity does not use the depth prepass to generate a copy of the depth buffer (this behavior is different in the Forward Rendering Path).
+
+If the Universal Renderer has the SSAO Renderer Feature, Unity executes the depth and normal prepass. SSAO uses the screen-space normal buffer to calculate ambient occlusion.
```

Comment:
SSAO uses the screen-space depth and normal buffers to calculate ambient occlusion.

Reply:
Done.